### ISSUE TITLE
My Sites: VoiceOver says "No speakable content"

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#12256]
Reported in p1564153937013200-slack-inclusion. 

## Expected behavior

VoiceOver should read all the elements on My Sites.

## Actual behavior

VoiceOver says "No speakable content could be found on the screen".

### Steps to reproduce the behavior

1. Tap My Sites 
2. Swipe down with 2 fingers 
3. No speakable content

Reproduced on WP 12.7 and 12.9. iPhone 7+ with iOS 12.3.1.

**Comment by shiki** [comment]
@designsimply This is still valid. Though I'm not sure if there are new designs/features affecting My Sites nowadays. If so, then maybe we should close this for now.

**Comment by rohanrk** [comment]
I'm not sure if this was addressed but I just tested on my iPad and looks like this works properly. 

Though there does seem to be an issue where Voiceover will try to read the disabled detail view on the right. That is iPad specific though.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
